sephora/_DEFINES/overmap.dm

```diff
@@ -1,17 +0,0 @@
-//Weapon modes
```

Comment:
You deleted this?

Reply:
Nope. Not sure why it shows that the entire file was remade eve though I didn't touch it. Best guess is that it's a quirk from renaming the directory.

Reply:
For reference, the "new file" is seen a bit further up. 

![image](https://user-images.githubusercontent.com/26130695/64029541-6e2c0180-cb0a-11e9-93e7-4a23f3c1082c.png)